agenta-backend/Dockerfile

```diff
@@ -17,6 +17,7 @@ RUN touch /app/agenta_backend/__init__.py
 # Project initialization:
 RUN poetry config virtualenvs.create false \
     && poetry install --no-interaction --no-ansi
+
 # remove dummy module
 RUN rm -r /app/agenta_backend
```

Comment:
Don't we need to create a dockerfile where the backend is copied to be used in this case?

Reply:
Yes, a dockerfile that is production ready.